content/aips/3D16C7B29565-AIP-FEI-RESERVE-FACTOR-UPDATE.md

```diff
@@ -0,0 +1,60 @@
+---
+title: FEI Reserve Factor Update
+status: Proposed
+author: Llama, Shreyas Hariharan, Rajath Alex
+shortDescription: FEI Reserve Factor Update
+discussions: https://governance.aave.com/t/bgd-aave-v2-ethereum-fei-security-report/10251
+created: 2022-10-18
+---
+
+# Simple Summary
+
+This proposal updates FEI’s Reserve Factor on Aave v2 Ethereum from 100% to 99%. As noted by [BGD](https://governance.aave.com/t/bgd-aave-v2-ethereum-fei-security-report/10251), the current configuration of FEI’s Reserve Factor at 100% does not accrue variable borrow debt and does not redirect the Reserve Factor fees to the Collector smart contract.
+
+# Abstract
+
+As noted [here](https://governance.aave.com/t/bgd-aave-v2-ethereum-fei-security-report/10251), the Reserve Factor (RF) on Aave is a percentage representing the proportion of the supplier’s yield that gets redirected to the protocol as a “fee”. For example, WETH on Aave v2 Ethereum has a 10% RF, so if the yield for suppliers would be 5% annually, 10% of it (0.5%) would go to the protocol itself, to a Collector contract and in the shape of aTokens.
```

Comment:
Are you 100% sure this sentence makes sense? If you get 5% yield you get 5% yield not 4.5% - the Rf is applied before. Would probably just remove the example.

Reply:
This is actually taken from the BGD write-up: https://governance.aave.com/t/bgd-aave-v2-ethereum-fei-security-report/10251#context-2

But if the example doesn't make sense, i'll just remove it for clarity's sake.

Reply:
Done